src/amo/reducers/users.js

```diff
@@ -22,6 +22,8 @@ export const LOAD_USER_ACCOUNT: 'LOAD_USER_ACCOUNT' = 'LOAD_USER_ACCOUNT';
 export const DELETE_USER_PICTURE: 'DELETE_USER_PICTURE' = 'DELETE_USER_PICTURE';
 export const FETCH_USER_NOTIFICATIONS: 'FETCH_USER_NOTIFICATIONS' = 'FETCH_USER_NOTIFICATIONS';
 export const LOAD_USER_NOTIFICATIONS: 'LOAD_USER_NOTIFICATIONS' = 'LOAD_USER_NOTIFICATIONS';
+export const DELETE_USER_ACCOUNT: 'DELETE_USER_ACCOUNT' = 'DELETE_USER_ACCOUNT';
+export const REMOVE_USER_ACCOUNT: 'REMOVE_USER_ACCOUNT' = 'REMOVE_USER_ACCOUNT';
```

Comment:
@bobsilverberg would you have an idea on how to improve this? "DELETE" comes from the user action (and also from the API call), so I like to have a `DELETE_USER_ACCOUNT` dispatched when user wants to remove their profile.

Yet, I also need an action to remove the user data from the state when the deletion has completed, hence the `REMOVE` because I want to remove the user data.

Given that we use `LOAD` to add a user to the state, can we maybe use `UNLOAD`?

Reply:
Hmm, `UNLOAD` does make sense, as we're unloading it from state. I was also going to say that maybe it could be xxx_USER_DATA as opposed to xxx_USER_ACCOUNT, because we're removing data from state, as opposed to removing an actual account, but yes, `UNLOAD` probably makes the most sense in this context.